Modules/Sources/WordPressUI/Views/Acknowledgements/AcknowledgementsListView.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+public struct AcknowledgementsListView: View {
+
+    @ObservedObject
+    var viewModel: AcknowledgementsListViewModel
+
+    public init(viewModel: AcknowledgementsListViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        Group {
+            if let error = viewModel.error {
+                EmptyStateView(error.localizedDescription, systemImage: "exclamationmark.triangle.fill")
+                    .multilineTextAlignment(.center)
+            } else if viewModel.isLoadingItems {
+                ProgressView()
+            } else {
+                List(viewModel.items) { item in
+                    NavigationLink {
+                        AcknowledgementsDetailView(item: item)
+                    } label: {
+                        VStack(alignment: .leading) {
+                            Text(item.title)
+                                .font(.body)
+                            if let description = item.description {
+                                Text(description)
+                                    .font(.caption)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        .onAppear(perform: viewModel.onAppear)
+        .onDisappear(perform: viewModel.onDisappear)
+        .navigationTitle(Localizations.acknowledgementsTitle)
+    }
+
+    struct Localizations {
```

Comment:
The common approach the app uses is to put these at the bottom of the file (but probably before the `#Preview`), and make it a private enum named `Strings`. The enum with no cases and that can't be initialized is the closest thing to a namespace.

```swift
private enum Strings {
   // .. 
}

```

Reply:
Ah yep thanks – sorted in cc7eb7c3d11f290b50b1285a97e621f1ebbff166